code/_expts/slotnick_etal_2001/analyze/plots/de_FigurizerCC.m

```diff
@@ -0,0 +1,42 @@
+%% Still needed:
```

Comment:
Please put the function definition at the top of a file, and comments below the function definition. I believe MATLAB uses that style to auto-pull docstrings for functions.